cmd/kubeadm/app/util/version.go

```diff
@@ -132,8 +137,8 @@ func splitVersion(version string) (string, string, error) {
 }
 
 // Internal helper: return content of URL
-func fetchFromURL(url string) (string, error) {
-	client := &http.Client{Transport: netutil.SetOldTransportDefaults(&http.Transport{})}
+func fetchFromURL(url string, timeout time.Duration) (string, error) {
+	client := &http.Client{Timeout: timeout, Transport: netutil.SetOldTransportDefaults(&http.Transport{})}
```

Comment:
How about we use `getReleaseVersionTimeout` here directly?

Reply:
+1